docs/tomcat.md

```diff
@@ -0,0 +1,67 @@
+
+# `tomcat_system` and `tomcat_access` Logging Receivers
+
+## Configuration
+
+To configure a receiver for your tomcat system logs, specify the following fields:
+
+| Field                 | Default                           | Description |
+| ---                   | ---                               | ---         |
+| `type`                | required                          | Must be `tomcat_system`. |
+| `include_paths`       | `[/opt/tomcat/logs/catalina.out]` | A list of filesystem paths to read by tailing each file. A wild card (`*`) can be used in the paths; for example, `/var/log/apache*/*.log`.
+| `exclude_paths`       | `[]`                              | A list of filesystem path patterns to exclude from the set matched by `include_paths`.
+
+To configure a receiver for your tomcat debug logs, specify the following fields:
+
+| Field                 | Default                                         | Description |
+| ---                   | ---                                             | ---         |
+| `type`                | required                                        | Must be `tomcat_debug`. |
+| `include_paths`       | `[/opt/tomcat/logs/localhost_access_log.*.txt]` | The log files to read. |
+| `exclude_paths`       | `[]`                                            | Log files to exclude (if `include_paths` contains a glob or directory). |
+
+Example Configuration:
+
+```yaml
+logging:
+  receivers:
+    tomcat_default_system:
```

Comment:
Can we remove "_default" from the sample config (The receiver ID would be used by Cloud Logging as the log name). It's better for users to use `tomcat_system` / `tomcat_access`.

Reply:
removed

Reply:
Thanks!